### PR TITLE
PR for competitions page

### PR DESCRIPTION
**Pull request by nath0002**

### CODE REVIEW

**Comment by fung-alvin** [comment]
Images can have rounded borders. I believe I used 30px radius for mine. I personally think there should be more spacing between either side of the text. For me, I limited the width of the text to be 90% of the screen. Also feel like there should be a bit more spacing between images and text. Mobile view looks neat.

File: public/competitionImage1.png
[the remark this answers is not in the document]
Comment (fung-alvin) [reply public/competitionImage1.png:1]: We can merge to master first and format layout later.

**Review by EgoEris — APPROVED** [review]
Quite happy with this! definitely need to do some minor css things like @fung-alvin said, rounding the image borders and potentially spacing between sections and the text inside the stat bubbles - however we'll take care of this post merge

**Review by fung-alvin — APPROVED** [review]
Will make minor adjustments to formatting after merging to master.

**Pull request merged**